modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/delegate/ParameterModelsLoaderDelegate.java

```diff
@@ -49,11 +49,11 @@ public ParameterModelsLoaderDelegate(Supplier<StereotypeModelLoaderDelegate> ste
         group.getDisplayModel().ifPresent(groupDeclarer::withDisplayModel);
         group.getLayoutModel().ifPresent(groupDeclarer::withLayout);
         group.getAdditionalModelProperties().forEach(groupDeclarer::withModelProperty);
-        group.getExclusiveOptionals().ifPresent(descriptor -> groupDeclarer
-            .withExclusiveOptionals(descriptor.getExclusiveOptionals(), descriptor.isOneRequired()));
         groupDeclarer.getDeclaration().setDescription(group.getDescription());
         group.getLayoutModel().ifPresent(groupDeclarer::withLayout);
       }
+      group.getExclusiveOptionals().ifPresent(descriptor -> groupDeclarer
+          .withExclusiveOptionals(descriptor.getExclusiveOptionals(), descriptor.isOneRequired()));
```

Comment:
This part needs to be checked. I had to change it because otherwise, the exclusive optionals wouldn't be added to the extension model for parameters that belong to the default group. But in the new Mule SDK all parameters got to the default group.
I'm not sure whether this could have an unexpected impact when using other SDK implementations. Also, it smells that I'm changing this only for the exclusive optionals... what about the other things?

Reply:
this change looks good to me. Java SDK doesn't offer a syntax for defining EO in the default group and XML SDK doesn't support the concept at all